### PR TITLE
devops: run BiDi tests

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

**Comment by yury-s** [comment]
Let's trigger the workflow only manually for now and apply other fixes that we discussed.

File: .github/workflows/tests_bidi.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    branches:
+      - main
+    paths:
+      - .github/workflows/tests_bidi.yml
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  FORCE_COLOR: 1
+  ELECTRON_SKIP_BINARY_DOWNLOAD: 1
```
Comment (yury-s) [inline .github/workflows/tests_bidi.yml:15]: remove?

**Review by yury-s — APPROVED** [review]

File: .github/workflows/tests_bidi.yml (hunk 1 above)
Comment (mxschmitt) [reply .github/workflows/tests_bidi.yml:15]: npm ci would still install it.

File: .github/workflows/tests_bidi.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/tests_bidi.yml
+  schedule:
+    - cron: '0 0 * * *'
```
[the remark this answers is not in the document]
Comment (mxschmitt) [reply .github/workflows/tests_bidi.yml:11]: Done

File: .github/workflows/tests_bidi.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    strategy:
+      fail-fast: false
+      matrix:
+        # TODO: add Firefox
+        channel: [bidi-chrome-beta]
+    steps:
+    - uses: actions/checkout@v4
+    - name: Install browser
+      run: |
+        if [[ ${{ matrix.channel }} == bidi-chrome-beta ]]; then
```
[the remark this answers is not in the document]
Comment (mxschmitt) [reply .github/workflows/tests_bidi.yml]: Done

**Pull request merged**